config/manager/manager.yaml

```diff
@@ -11,18 +11,18 @@ metadata:
   name: controller-manager
   namespace: system
   labels:
-    control-plane: controller-manager
+    control-plane: kubesaw-controller-manager
```

Comment:
TBH, I'm not sure if this labels is not used somewhere else (eg in sanbox-sre). Let's keep the label there and add a new one instead

Reply:
yes you are right, added new ones now.